networks/genesis-validators.md

```diff
@@ -35,7 +35,7 @@ Some important notes on joining as a genesis validator:
     of the Cosmos Hub from `cosmoshub-3` to `cosmoshub-4` will have
     recieve fairdrop OSMO. You can verify that a Cosmos address has
     received coins in the fairdrop by inputting an address here:
-    <https://airdrop.osmosis.zone/>.
+    <https://airdrop.osmosis.zone/>. # TODO: Is there an airdrop page anymore?
```

Comment:
Can we comment this out and mention that the link is broken but left for historic reasons?

Reply:
Added a note. I also created a [`.lycheeignore` file](https://github.com/lycheeverse/lychee-action/tree/master#excluding-links-from-getting-checked) to ignore some links from getting checked.

- `https://github.com/osmosis-labs/osmosis-ci*`, because it's a private repo and will always fail. Unsure if we can somehow provide GH token(s) so it can access the private repo, but ignoring that for now.
- `https://eips.ethereum.org/EIPS/eip-191*`, because it's reading the OpenAPI html file and confusing newlines `\n` with part of the link so it fails. Not sure if you know how we'd go about doing regex in the file-checker action to ignore specifically the OpenAPI file `client/docs/static/openapi/index.html`.
- `https://airdrop.osmosis.zone/`, as we're leaving it there.
- `http://skynet.paullovette.com/`, unsure what skynet and providers are so I couldn't find this either, but wanted to keep it in our documents to keep the thanks to them.

Reply:
Let me know if you'd rather not have the extra `.lycheeignore` file though!